### DISCUSSION TITLE
Nuova versione 1.5.9

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#744] [Annunci]
**Queste sono le modifiche presenti nella versione 1.5.9:**
- Modifica: aggiunte le proposte di voto per lo scrutinio del giudizio sospeso
- Fix: revisione procedura dello scrutinio del giudizio sospeso
- Fix: revisione procedura dello scrutinio rinviato
- Fix: revisione procedura di archiviazione documenti
- Fix: revisione procedura di passaggio al nuovo anno scolastico
- Nuovo: archivio degli avvisi degli anni precedenti (menu **STAFF > AVVISI > ARCHIVIO**)

Per quanto riguarda lo scrutinio sospeso, ora si deve inserire la proposta di voto prima di iniziare lo scrutinio. La proposta di voto corrisponde alla valutazione della verifica di recupero svolta dall'alunno.
E' anche possibile aggiungere un giudizio descrittivo sulla prova di recupero svolta dall'alunno, ma non è obbligatorio inserirlo. Tale giudizio, come la proposta di voto, rimane agli atti nel registro del docente, quando si procede all'archiviazione.

**Comment by luigirotondi** [comment]
Dopo aver effettuato lo scrutinio "giudizio sospeso" si può procedere al cambio d'anno?

**Reply by trinko** [reply]
> Dopo aver effettuato lo scrutinio "giudizio sospeso" si può procedere al cambio d'anno?

Si, ma prima bisogna eseguire la procedura di archiviazione, scaricando i documenti presenti nella cartella **FILES/archivio**.
**La procedura di passaggio al nuovo anno cancella i documenti presenti nell'archivio.**

**Reply by colellagabriele889** [reply]
Salve, segnalo un problema. Dopo aver effettuato il cambio d'anno abbiamo perso 10 studenti. Sono recuperabili in qualche modo?

**Reply by trinko** [reply]
> Salve, segnalo un problema. Dopo aver effettuato il cambio d'anno abbiamo perso 10 studenti. Sono recuperabili in qualche modo?

Gli utenti che vengono eliminati sono quelli disabilitati e quelli in uscita, cioè ammessi all'esame in quinta. Quindi se uno studente di quinta non passa l'esame risulterà rimosso. Se questo è il caso, l'unico modo di recuperarli è dal backup, anche se è un po' laborioso.

Se non sei in questa situazione, allora gli studenti non sono rimossi, ma può capitare che non siano stati associati a nessuna classe: in tal caso controlla che siano prensenti alla pagina  **ALUNNI > MODIFICA** selezionando il filtro **NESSUNA CLASSE**.
